hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestContainerAllocation.java

```diff
@@ -1392,4 +1412,15 @@ public void testContainerRejectionWhenAskBeyondDynamicMax()
 
     rm1.close();
   }
+
+  private CapacityScheduler getCapacityScheduler(MockRM rm1, int memoryGb) {
```

Comment:
The name getCapacityScheduler is misleading. A getter shouldn't set up clusterResources or init the queue structure. I would prefer if there was a separate getter and a mockCS reinit method, which sets up the resources/queues.

Reply:
I renamed the method to setupCapacitySchedulerI